[BUG] Compiler error while checking specifically only `eww`

## Checklist before submitting an issue
- [x] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [x] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [x] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

## Describe the bug
Compiler error while checking only the `eww` crate.

## Reproducing the issue
`cargo check -p eww`

## Expected behaviour
Successful check, no compiler errors.

## Additional context

<details>
<summary>
compiler output
</summary>

```
    Checking eww v0.2.0 (/home/zeta0/dev/eww/crates/eww)
error[E0432]: unresolved import `yuck::config::backend_window_options::WindowType`
  --> crates/eww/src/app.rs:18:9
   |
18 |         backend_window_options::WindowType,
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WindowType` in `config::backend_window_options`

error[E0432]: unresolved imports `yuck::config::backend_window_options::Side`, `yuck::config::backend_window_options::WindowType`
   --> crates/eww/src/display_backend.rs:106:34
    |
106 |         backend_window_options::{Side, WindowType},
    |                                  ^^^^  ^^^^^^^^^^ no `WindowType` in `config::backend_window_options`
    |                                  |
    |                                  no `Side` in `config::backend_window_options`

error[E0609]: no field `window_type` on type `BackendWindowOptions`
   --> crates/eww/src/app.rs:376:43
    |
376 |             if window_def.backend_options.window_type != WindowType::Normal {
    |                                           ^^^^^^^^^^^ unknown field

error[E0609]: no field `wm_ignore` on type `BackendWindowOptions`
   --> crates/eww/src/display_backend.rs:113:57
    |
113 |         let window_type = if window_def.backend_options.wm_ignore { gtk::WindowType::Popup } else { gtk::WindowType::Toplevel };
    |                                                         ^^^^^^^^^ unknown field

error[E0609]: no field `sticky` on type `BackendWindowOptions`
   --> crates/eww/src/display_backend.rs:121:39
    |
121 |         if window_def.backend_options.sticky {
    |                                       ^^^^^^ unknown field

error[E0609]: no field `struts` on type `BackendWindowOptions`
   --> crates/eww/src/display_backend.rs:158:56
    |
158 |             let strut_def = window_def.backend_options.struts;
    |                                                        ^^^^^^ unknown field

error[E0609]: no field `window_type` on type `BackendWindowOptions`
   --> crates/eww/src/display_backend.rs:211:52
    |
211 |                 &[match window_def.backend_options.window_type {
    |                                                    ^^^^^^^^^^^ unknown field

Some errors have detailed explanations: E0432, E0609.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `eww` due to 7 previous errors
```

</details>

--- a/crates/eww/Cargo.toml
+++ b/crates/eww/Cargo.toml
@@ -11,8 +11,8 @@ homepage = "https://github.com/elkowar/eww"
 
 [features]
 default = ["x11"]
-x11 = ["gdkx11", "x11rb"]
-wayland = ["gtk-layer-shell", "gtk-layer-shell-sys"]
+x11 = ["gdkx11", "x11rb", "yuck/x11"]
+wayland = ["gtk-layer-shell", "gtk-layer-shell-sys", "yuck/wayland"]
 [dependencies.cairo-sys-rs]
 version = "0.14.0"
 

